flexeval/core/reward_model/sequence_classification.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from typing import Any
+
+import torch
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from flexeval.core.reward_bench_dataset import RewardBenchInstance
+from flexeval.core.reward_model.base import RewardModel
+from flexeval.utils.hf_utils import get_default_model_kwargs
+
+
+class SequenceClassificationRewardModel(RewardModel):
+    """Pairwise judge using a chat language model to compare two model or human
+    outputs.
+    """
+
+    def __init__(
+        self,
+        model: str,
+        model_kwargs: dict[str, Any] | None = None,
+        tokenizer: str | None = None,
+        tokenizer_kwargs: dict[str, Any] | None = None,
+    ) -> None:
+        tokenizer = tokenizer if tokenizer else model
+        tokenizer_kwargs = tokenizer_kwargs or {}
+        self.tokenizer = AutoTokenizer.from_pretrained(tokenizer, **tokenizer_kwargs)
+
+        model_kwargs = get_default_model_kwargs(model_kwargs)
+        self.model = AutoModelForSequenceClassification.from_pretrained(model, **model_kwargs)
+        self.model.eval()
+
+        self.tokenizer = AutoTokenizer.from_pretrained(model)
```

Comment:
Duplicate to the line 27. Typo?

Reply:
Good catch!
Thank you, fixed.

https://github.com/sbintuitions/flexeval/pull/101/commits/3be1c8e520b86f408714025b4fe2fafa7e6281d6